hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/FederationInterceptorREST.java

```diff
@@ -736,6 +726,9 @@ public AppsInfo getApps(HttpServletRequest hsr, String stateQuery,
     // HttpServletRequest does not work with ExecutorCompletionService.
     // Create a duplicate hsr.
     final HttpServletRequest hsrCopy = clone(hsr);
+
+
```

Comment:
Needed three break lines?

Reply:
Thank you for your help to review the code, I will modify the code. This part of the code, I need to continue to improve.